README.md

```diff
@@ -1,6 +1,9 @@
 ## k8s-cleanup
 
-Cleans up exited containers and dangling images/volumes running as a DaemonSet (`deploy-ds.yml`) and cleans up old replica sets and finished jobs as a CronJob (`deploy-cron.yml`).
+Here are 3 cleanups you can apply on your kubernetes cluster:
+* Cleans up exited containers and dangling images/volumes running as a DaemonSet (`deploy-ds.yml`).
```

Comment:
If you're calling the last `etcd-empty-dir-cleanup.yml`, might be worth renaming `deploy-ds.yml` and `deploy-cron.yml` to `docker-clean.yml` and `k8s-clean.yml` ?